### PR TITLE
Long factorial

### PR DESCRIPTION
**Pull request by Saviour1001**
Added the code for Long Factorials in C++.
This PR closes the issue #362

### CODE REVIEW

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
add atleast 2 testcases and time and space complexity. also link it to the folder.

File: Array/readme.md [hunk 1]
```diff
@@ -12,19 +12,22 @@ The syntax is:
...
 
- * Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) 
- * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp) 
- * Repeating and Missing Number ----> [C++](/Code/C++/repeating_and_missing_number.cpp) 
+- Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp)
+- Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp)
+- Repeating and Missing Number ----> [C++](/Code/C++/repeating_and_missing_number.cpp)
 
+* Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) | [Java](/Code/Java/insertion_at_start.java) | [Python](/Code/Python/insertion_at_start.py)
+* Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp)
```
Comment (OjusWiZard) [inline Array/readme.md:32]: Index only those codes that this PR is supposed to add.

**Review by OjusWiZard — CHANGES_REQUESTED** [review]

File: Array/readme.md (hunk 1 above)
Comment (Saviour1001) [reply Array/readme.md:32]: The VS Code's inbuild Markdown formatter does that changes to improve the spaces in all the lines for a uniform format through out the Readme file. The added line is just one , rest formatting is done by VS code just to make it uniform.

**Comment by tarun26091999** [comment]
resolve the conflict.

**Review by tarun26091999 — APPROVED** [review]

**Pull request merged**